src/components/pages/dashboard/withdraw.js

```diff
@@ -1,10 +1,106 @@
-import React from "react";
+import React, { useState } from "react";
 import DashboardLayout from "../../layout/dashboard.layout";
+import * as yup from "yup";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useNavigate } from "react-router-dom";
+import {
+  getUserByAccountNumber,
+  getAllUsers,
+  getUserCurrentBalance,
+  getUserIndexByAccountNumber,
+  getLoggedInUserAccountNumber,
+} from "../../../helpers/user.helper";
 
 export const Withdraw = () => {
+  const navigate = useNavigate();
+  const [inputValue, setInputValue] = useState();
```

Comment:
This is not needed. useForm hook is registering and handling the input changes.

Reply:
Resolved